en/components/grids_templates/excel_style_filtering.md

```diff
@@ -0,0 +1,246 @@
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="tree-grid-sample-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+}
+@@if (igxName === 'IgxHierarchicalGrid') {
+<div class="sample-container loading" style="height:700px">
+    <iframe id="hierarchical-grid-sample-1-iframe" src='{environment:demosBaseUrl}/hierarchical-grid/hierarchical-grid-excel-style-filtering' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="hierarchical-grid-sample-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+}
+
+<div class="divider--half"></div>
+
+###Usage
+
+To turn on the excel style filtering, two inputs should be set. The [`allowFiltering`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#allowfiltering) should be set to `true` and the [`filterMode`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#filtermode) should be set to `excelStyleFilter`.
+
+@@if (igxName === 'IgxGrid') {
+```html
+<igx-grid [data]="data" [autoGenerate]="true" [allowFiltering]="true" [filterMode]="'excelStyleFilter'">
+</igx-grid>
+```
+}
+@@if (igxName === 'IgxTreeGrid') {
+```html
+<igx-tree-grid [data]="data" [autoGenerate]="true" [allowFiltering]="true" [filterMode]="'excelStyleFilter'">
+</igx-tree-grid>
+```
+}
+@@if (igxName === 'IgxHierarchicalGrid') {
+```html
+<igx-hierarchical-grid [data]="data" [autoGenerate]="true" [allowFiltering]="true" [filterMode]="'excelStyleFilter'">
+</igx-hierarchical-grid>
+```
+}
+
+###Interaction
+
+In order to open the filter menu for a particular column, the filter icon in the header should be clicked. If the column can be sorted, pinned, moved or hidden along with the filtering functionality there will be buttons for the features that are turned on.
+
+If no filter is applied all the items in the list will be selected. They can be filtered from the input above the list. In order to filter the data you can select/deselect the items in the list and click Apply button. The filtering applied throught the list items create filter expressions with `equals` operator and the logic operator between each expression is `OR`. If you want to clear the filter you can check Select All item and then Apply button. 
+
+To apply filter with different expressions you can click the **Text filter**, which will open a sub menu with all available filter operators for the particular column. Selecting one of them will open the custom filter dialog, where you can add as many expressions as you want with different filter and logic operators. There is also a clear button which can clear the filter.
```

Comment:
missing comma - To apply **a** filter with different expressions, you can click
There is also a clear button, which can clear the filter.